benches/pairhmm.rs

```diff
@@ -73,6 +79,9 @@ impl GapParameters for SemiglobalGapParams {
     }
 }
 
+#[derive(
+    Default, Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug, Serialize, Deserialize,
+)]
```

Comment:
I think change in benches isn't realy necessary this struct are create only for benchmark, isn't realy public API

Reply:
Good catch! I missed this when the script added these derives everywhere! I'll remove those now!